test/unit/reports/get-events.test.js

```diff
@@ -0,0 +1,76 @@
+const { PAYMENT_EVENT } = require('../../../app/constants/event-types')
+const { stringifyEventData } = require('../../helpers/stringify-event-data')
+
+jest.mock('../../../app/storage')
+const { getClient: mockGetClient, odata: mockOdata } = require('../../../app/storage')
+
+const mockListEntities = jest.fn()
+
+const mockTableClient = {
+  listEntities: mockListEntities
+}
+
+let extractedEvent
+let enrichedEvent
+let events
+
+const { getEvents } = require('../../../app/reports/mi-report/get-events')
```

Comment:
Maybe group this with 'PAYMENT_EVENT' and 'stringifyEventData'?

Reply:
Kept separate so it stands out